docs/guides/modules/security/pages/audit-logs.adoc

```diff
@@ -1,22 +1,238 @@
+* IAM role or user with appropriate S3 write permissions
+* Network configuration to allow secure connections between CircleCI and AWS
+====
+
+The following steps describe how to set up AWS prerequisites:
+
+. Create an S3 bucket in your AWS account to store the audit logs.
++
+You can also specify an optional folder or prefix (for example, `my-bucket/audit-logs`).
++
+Take note of the AWS region where you created the S3 bucket (for example,`us-east-1`).
+. Create an IAM role that CircleCI can assume. This role requires two key policies:
+.. Trust policy
++
+This policy allows CircleCI's service to assume the role. The principal ARN in this policy is a template that you must configure with your specific AWS account and CircleCI organization IDs.
++
+The complete template for the trust relationship is the following:
++
+[source,json]
+----
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Principal": {
+                "Federated": "arn:aws:iam::<aws-account-id>:oidc-provider/oidc.circleci.com/org/<org-id>"
+            },
+            "Action": "sts:AssumeRoleWithWebIdentity",
+            "Condition": {
+                "StringEquals": {
+                    "oidc.circleci.com/org/<org-id>:aud": "<org-id>"
+                }
+            }
+        }
+    ]
+}
+----
++
+Replace `<aws-account-id>` with your 12-digit AWS account ID and `<org-id>` with your unique CircleCI organization ID (found on the menu:Organization Settings[Overview] page in the link:https://app.circleci.com/[CircleCI web app]).
+
+.. Permissions policy
++
+This policy grants the role permission to write objects to your S3 bucket (and any specified prefix).
++
+The minimum required access policy for the role is as follows:
++
+[source,json]
+----
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Action": [
+                "s3:PutObject",
+                "s3:GetObject",
+                "s3:ListBucket"
+            ],
+            "Resource": [
+                "arn:aws:s3:::<bucket-name>/*",
+                "arn:aws:s3:::<bucket-name>"
+            ]
+        }
+    ]
+}
+----
++
+Replace `<bucket-name>` with the name of your S3 bucket.
+
+. Set up an OIDC identity provider to allow CircleCI's OIDC tokens to be used for authentication.
+.. Navigate to the IAM Console in AWS.
+.. In the left navigation menu, select *Identity Providers*.
+.. Select *Add provider*.
+.. Choose *OpenID Connect* as the provider type.
+.. In the *Provider URL* field, specify `\https://oidc.circleci.com/org/<org-id>` (replace `<org-id>` with your CircleCI organization ID).
+.. In the *Audience* field, enter your organization ID.
+
+
+[#configure-audit-log-streaming]
+=== Configure Audit Log Streaming
+
+. Navigate to menu:Organization Settings[Security].
+. Scroll down to the **Audit Logs > Streaming audit logs** section and select btn:[Stream audit logs].
```

Comment:
you can use the menu:[] macro throughout. The > looks a little different with and without
<img width="155" height="82" alt="Screenshot 2025-09-08 at 12 10 41" src="https://github.com/user-attachments/assets/30c4b1de-c21b-443c-8201-f6a44b9aea3b" />